src/schema/nmdc.yaml

```diff
@@ -791,15 +791,18 @@ slots:
 
   eluent_introduction_category:
     range: EluentIntroductionCategoryEnum
-    description: how a processed sample is introduced into a mass spectrometer, for example liquid chromatography or direct infusion through a syringe
+    description: A high-level description for how the processed sample is introduced into a mass spectrometer.
+    examples:
+      - value: liquid_chromatography
+      - value: direct_infusion_syringe
 
   has_mass_spectrometry_configuration:
     range: MassSpectrometryConfiguration
-    description: a description of how the mass spectrometry was configured
+    description: An identifier for the associated MassSpectrometryConfiguration.
```

Comment:
Yeah, "identifier" is better wording than "description". I recommend "A link to..." or "The identifier of..."